workspace/src/localization/localization_py/localization_py/state_estimation.py

```diff
@@ -1,190 +1,240 @@
+import csv
 import rclpy
 from rclpy.node import Node
 from art_msgs.msg import VehicleState
-#from nav_msgs.msg import 
 from sensor_msgs.msg import Imu, NavSatFix, MagneticField
-#from sensor_msgs.msg import NavSatFix
+from chrono_ros_msgs.msg import ChVehicle
 from ament_index_python.packages import get_package_share_directory
-
 import matplotlib.pyplot as plt
 import matplotlib
 import math
 import numpy as np
-
-
 import sys
 import os
 
-
-#TODO: what does this do?
 ament_tools_root = os.path.join(os.path.dirname(__file__), '.')
 sys.path.insert(0, os.path.abspath(ament_tools_root))
```

Comment:
I don't think this is needed anymore, right?

Reply:
This is needed to import the EKF, Particle Filter, and coordinate transfer... should those also be passed in as arguments? I believe this is the way it is how it is done in perception as well, but I may be mistaken...

Reply:
oh interesting, I would think this is supported by ROS with something like `package_name.<file>`. I'll look into it.

Reply:
Ya I think if you do [this line](https://github.com/uwsbel/autonomy-research-testbed/blob/8dfad7351cc1ed8a3ca161c2015a0ec22ad13a04/workspace/src/common/launch/launch_utils/setup.py#L9) in `localization_py` and then do `localization_py.<file>` it should work without needing these two lines.